custom/reach/reports/reports.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.urls import reverse
+
+from corehq.apps.locations.permissions import location_safe
+from corehq.apps.reports.standard import CustomProjectReport
+
+
+@location_safe
+class DashboardReport(CustomProjectReport):
+    slug = 'dashboard_report'
+    name = 'REACH Dashboard'
+
+    @classmethod
+    def get_url(cls, domain=None, **kwargs):
+        return reverse('program_overview', args=[domain])
+
+    @classmethod
+    def show_in_navigation(cls, domain=None, project=None, user=None):
+        #TODO replace to new feature flag
+        return True
```

Comment:
```suggestion
        return False
```

Reply:
Hiding this for now just in case we need to

Reply:
:+1:  I added a new feature flag for this, so we can enable/disable reports from UI without new PR.